python_modules/dagster-webserver/dagster_webserver/webserver.py

```diff
@@ -274,7 +274,9 @@ def build_static_routes(self):
     def build_routes(self):
         routes = (
             [
-                Route("/dagit_info", self.dagit_info_endpoint),
+                Route("/dagster-webserver-info", self.webserver_info_endpoint),
```

Comment:
nit: maybe `server_info` ? this feels like a lot

Reply:
yeah good idea